package.json

```diff
@@ -11,6 +11,9 @@
 		"type": "git",
 		"url": "https://github.com/dojo/loader.git"
 	},
+	"dependencies": {
+		"dojo-core": "dojo/core"
```

Comment:
We've been discussing this in #dojo on IRC and myself, @neonstalwart, and @kitsonk are inclined to think that this plugin would be appropriate in another repo, in which case it would be okay to rely on core.  (Personally I can't even justify it being in core, because we may end up not needing to rely on the text plugin nearly as heavily as we do in Dojo 1.)

Meanwhile, I kind of thought one of the motives of this task was to test the loader plugin API, and IIRC it was suggested that the text plugin would only go so far to test that anyway.  This PR seems to be focused centrally on testing the plugin itself, with testing the loader's plugin API as a side effect at most.  As it stands, this PR could likely collectively be migrated to another repo for the most part if we wanted.

Were we expecting to have a separate PR for more exhaustive/direct plugin API tests, or was that expected to be tackled here?